pubspec.yaml

```diff
@@ -94,6 +94,7 @@ dependencies:
   workmanager:
     git:
       url: https://github.com/breez/flutter_workmanager.git
+  csv: ^5.0.2
```

Comment:
dependencies ares alphabetically ordered, please put this after `clipboard_watcher`

Reply:
Thanks for letting me know. Done.